ibis/backends/tests/test_map.py

```diff
@@ -227,8 +227,8 @@ def test_literal_map_get_broadcast(backend, alltypes, df):
         param(["a", "b"], ["1", "2"], id="int"),
     ],
 )
-@pytest.mark.notimpl(
-    ["flink"], raises=AssertionError, reason="WIP; got [('a', 1), ('b', 2)] instead"
+@pytest.mark.notyet(
+    ["flink"], raises=AssertionError, reason="got list of tuples instead"
```

Comment:
Can we add a short summary (either here or in the commit message) about how this is blocked by `pyflink`'s upper bound on `pyarrow`?

Reply:
Done, if that's sufficient (I assume can look at the blame then and pull out the comments from the PR).